sci-biology/salmon/salmon-0.6.0.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```

Comment:
Thank you all for the comments. `portage-2.2.25` just made it to prefix yesterday so I'll try to move to `EAPI=6`.